src/main/java/io/seqera/migtool/Driver.java

```diff
@@ -0,0 +1,29 @@
+package io.seqera.migtool;
+
+public enum Driver {
+    MYSQL("com.mysql.cj.jdbc.Driver"),
+    H2("org.h2.Driver"),
+    SQLITE("org.sqlite.JDBC"),
+    POSTGRES("org.postgresql.Driver"),
+    TCPOSTGRES("org.testcontainers.jdbc.ContainerDatabaseDriver");
```

Comment:
As far as I can see, this driver initializes a container database directly behind the scenes just by using a `jdbc:tc:...` connection URL.

Given that we already take care of starting a TC container independently in tests, I think it might not be worth adding support for it. Do you think it's worth?

Reply:
It is probably not needed, I can delete it.